src/main/resources/application.yml

```diff
@@ -13,7 +13,17 @@ server:
 oauth:
   clientId: ${CLIENT_ID:9fbd4ae2-2db3-44c7-9544-88e80255b56e}
   clientSecret: ${CLIENT_SECRET:test}
-  checkTokenEndpointUrl: ${CHECK_TOKEN_URL:http://localhost:8080/oauth/check_token}
+  serviceUrl: ${OAUTH_SERVICE_URL:http://localhost:8080}
```

Comment:
DOD - You have changed a critical environment value but haven't updated documentation or terraform templates.

Reply:
NO.  FULL CHANGE IS :

 serviceUrl: ${OAUTH_SERVICE_URL:http://localhost:8080}
  checkTokenEndpointUrl: "${oauth.serviceUrl}/oauth/check_token"

It's exactly the same

Reply:
Point is not the values but the change of key ENV variables without documentation update.

Reply:
NO Env variables have been changed ...
Added OAUTH_SERVICE_URL to readme

Reply:
@robertmarks do you have the terraform changes for this?

Reply:
The changes have already been merged into Terraform config